examples/plain-html/index.html

```diff
@@ -80,7 +104,7 @@
     })
     dialog.addEventListener('close', () => dialogButton.isDisabled = false)
   </script>
-  <div class="component-container">
+  <leo-option class="component-container">
```

Comment:
this doesn't seem right...?

Reply:
Wow, it was not :laughing: - I installed an extension for automatically editing the closing tag. I guess it has some bugs :laughing: 

Interestingly, it didn't affect the display at all - I guess the HTML parser is **really** forgiving